webpack/assets/javascripts/foreman_tools.js

```diff
@@ -36,17 +35,10 @@ export function activateDatatables() {
   });
 }
 
-export function userInheritedRolesUpdater(id) {
-  $('#roles_tab li').hide();
-  $("#roles_tab li[data-id = '" + id + "']").show();
-  $('.dropdown-menu li a').click(function () {
-    $(this).parents('.dropdown').find('.btn').html($(this).text() + ' <span class="caret"></span>');
-  });
-}
-
-export function activateTooltips(el = $('body')) {
+export function activateTooltips(el = 'body') {
+  el = $(el);
   el.find('[rel="twipsy"]').tooltip({ container: 'body' });
-  el.find('.ellipsis').tooltip({ container: 'body', title: () => {
+  el.find('.ellipsis').tooltip({ container: 'body', title: function () {
```

Comment:
We can't use `=>` here because we want `this` to point at the object the tooltip is added to - not the tools module.